src/stores/errorStore.ts

```diff
@@ -22,14 +23,14 @@ export class ErrorStore {
     }
 
     @action
-    public addError(message: string, error?: Error) {
+    public addError(message: string, error?: IError) {
```

Comment:
Hmm, error is always given as param. I would change: error? -> error, error as required param

Reply:
Hmm, really? That would remove the possibility for us to create simple error notification.
Instead we would always need to come up with an exception.

Reply:
Check https://github.com/HSLdevcom/jore-map-ui/blob/20c626455a28815a44765c6a97e11a1cb27c6656/src/services/routePathLinkService.ts#L79